.github/workflows/links.yml

```diff
@@ -0,0 +1,28 @@
+name: GitHub Markdown Links
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 8 * * 6'
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - run: sudo apt-get update -y
+      - run: >-
+          sudo apt-get install -y
+          pandoc
+      - uses: actions/checkout@v3
+      - run: |
+          find . -name '*.html' -delete
+      - run: |
+          find . -name '*.md' -exec pandoc -i {} -o {}.html \;
+      - uses: anishathalye/proof-html@v1
+        with:
+          directory: .
+          check_favicon: false
+          empty_alt_ignore: true
+          url_ignore_re: |
+            ^https:\/\/docs\.github\.com\/
+            ^https:\/\/github\.com\/cleanlab\/cleanlab\/projects
```

Comment:
```
^https:\/\/docs\.github\.com\/
^https:\/\/github\.com\/cleanlab\/cleanlab\/projects
```
can remove these 2 lines, they are related to other project

Reply:
Hmm Noted.